twitter_like_app/src/components/home_component/Home.jsx

```diff
@@ -0,0 +1,43 @@
+import React,{Component} from "react";
```

Comment:
No sense to name it `HomeComponent`. It should be `Home` in this case. 
+ The name of the directory should be the same as the name of component. `home_component` -> `Home` - no good. 
+ It looks like this is the page, not just simple component. So you can create directory `pages` in the `src` directory and have there only pages like: 'Home', 'Profile', 'VerifyEmail' etc...